### ISSUE TITLE
In "manage access" page, the status of organization check boxes' is not the reflection of the real situation

### ISSUE THREAD

**Issue by jiangweiyi** [servicecatalog/development#261]
Date: 2016/07/22
[Test category]: creative Test

Severity: minor

[Steps to reproduce]
 1. Create a marketplace, set a supplier as marketplace owner of it;
 2. Login as the supplier, click on "manage access", check "Authorized customers only", select himself to have access to it;
 3. Supplier publish and activate a service on the marketplace;
 4. Go to "Activate or Deactivate services" page to deactivate this service;
 5. Go to "manage access" page, to unselect himself to have access to it;
 6. Go to  "Activate or Deactivate services" page to activate this service;
 7. Go to "manage access" page to check the status of supplier;
 8. Go to "go to marketplace" page to check the marketplace in dropdown list;

[Expected result]
After step7 and step8, the status should be the same, the supplier should not checked and can not go to the marketplace.

[Observed result]
After step7, the supplier was selected and unclickable in this marketplace.
After step8, the marketplace are not shown in the dropdown list, and supplier can not login to this marketplace.

**Comment by MiethanerR** [comment]
this will be fixed together with issue #215

**Comment by MiethanerR** [comment]
issue #215 was merged into master

**Comment by MiethanerR** [comment]
merged fix for issue #215 into master

**Comment by SylviaDehn** [comment]
[Build] 2016/08/09 verified
